dotnetAvaloniaFrameBuffer/__change__.csproj

```diff
@@ -16,15 +16,15 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Avalonia" Version="11.0.11" />
-        <PackageReference Include="Avalonia.Skia" Version="11.0.11" />
-        <PackageReference Include="Avalonia.LinuxFramebuffer" Version="11.0.11" />
-        <PackageReference Include="Avalonia.Desktop" Version="11.0.11" />
-        <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.11" />
-        <PackageReference Include="Avalonia.Fonts.Inter" Version="11.0.11" />
+        <PackageReference Include="Avalonia" Version="11.3.7" />
+        <PackageReference Include="Avalonia.Skia" Version="11.3.7" />
+        <PackageReference Include="Avalonia.LinuxFramebuffer" Version="11.3.7" />
+        <PackageReference Include="Avalonia.Desktop" Version="11.3.7" />
+        <PackageReference Include="Avalonia.Themes.Fluent" Version="11.3.7" />
+        <PackageReference Include="Avalonia.Fonts.Inter" Version="11.3.7" />
```

Comment:
Did you run the tests? These are changes outside the scope of the TIE-1063.

Reply:
Yes